womtool/src/test/scala/womtool/WomtoolValidateSpec.scala

```diff
@@ -1,64 +1,105 @@
 package womtool
 
+import java.nio.file.Path
+
 import better.files.File
 import cromwell.core.path.DefaultPathBuilder
 import org.scalatest.{FlatSpec, Matchers}
 import womtool.WomtoolMain.{SuccessfulTermination, UnsuccessfulTermination}
 
+import scala.collection.immutable
+
 class WomtoolValidateSpec extends FlatSpec with Matchers {
 
+  private def httpImportTestCaseImport(language: String) =
```

Comment:
Rather than have to do all this, I suspect this would be easier and neater if you reverted the changes in this file and modeled these tests more after the `WomtoolInputsSpec` where the logic is:
* For each directory in the `inputs` directory
    * run womtool with some parameters (in that case `womtool inputs`) on the equivalent file in the `validate` directory
    * check that the command line output matches the expectation.

It might be easy enough to duplicate that logic (or make it generalizable) to be able to say:
* For each directory in a new `validate-with-imports` directory
    * run `womtool validate` with a `-l` parameter against the equivalent file in the `validate` directory
    * check that the command line output matches an expectation

Reply:
You suspected correctly! It was definitely neater! I refactored them based on this suggestion. Thank you!